src/main/java/com/googleinterns/zoomtube/servlets/LectureServlet.java

```diff
@@ -163,7 +161,8 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
 
   @VisibleForTesting
```

Comment:
since it is now public we can remove this @visible for testing

Reply:
good catch, removed!